images/mod-tile/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pyyaml
+psycopg2
+requests
+mapcoloniesjsonlogger==0.0.14
```

Comment:
we should use strict deps versions, in addition, I don't think we need pyyaml

Reply:
done